src/views/blocks/rsvp/details/availability.php

```diff
@@ -54,7 +54,7 @@
 		<?php if ( $show_unlimited) : ?>
 			<span class="tribe-block__rsvp__unlimited"><?php echo esc_html( $handler->unlimited_term ); ?></span>
 		<?php endif; ?>
-	<?php elseif ( $remaining_tickets <= $threshold ) : ?>
+	<?php elseif ( 0 === $threshold || $remaining_tickets <= $threshold ) : ?>
```

Comment:
I think I see what you are doing here - "set it to 0 and it never shows"?

Reply:
We absint() the threshold now, so 0 or blank in the TEC settings wouldn't work